lib/mayaUsd/fileio/utils/readUtil.cpp

```diff
@@ -414,6 +417,14 @@ bool UsdMayaReadUtil::SetMayaAttr(
         MPlug& attrPlug,
         const VtValue& newValue)
 {
+    SdfValueTypeName plugValueType = Converter::getUsdTypeName(attrPlug,false);
```

Comment:
Similar comment as elsewhere about potentially needing to plumb `translateMayaDoubleToUsdSinglePrecision` through here, or otherwise put it in a `ConverterArgs`.